src/features/search/components/__tests__/SearchResults.native.test.tsx

```diff
@@ -82,7 +82,7 @@ describe('SearchResults component', () => {
       expect(queryByTestId('locationButton')).toBeTruthy()
     })
 
-    it('should redirect on filters page on location button click', async () => {
+    it('should update the temporary search state by actual search state', async () => {
```

Comment:
je trouve que la recherche est déjà très complèxe

bien que `temporary` me semble plus explicite et approprié, partout ailleurs ça se nomme `staged`

je pense que ça risque de créer de la confusion si on introduit un synonyme

---

mon niveau d'anglais est pas terrible, je n'ai pas compris le `by`

je propose :

```suggestion
    it('should update the staged search state with the actual search state', async () => {
```